tutorials/Brier_Score.ipynb

```diff
@@ -1,131 +1,2206 @@
+                        ],
+                        "text/plain": [
+                            "<xarray.DataArray (threshold: 4)>\n",
+                            "array([0.    , 0.2532, 0.2532, 0.    ])\n",
+                            "Coordinates:\n",
+                            "  * threshold  (threshold) float64 -1.0 0.5 1.0 3.0"
+                        ]
+                    },
+                    "execution_count": 20,
+                    "metadata": {},
+                    "output_type": "execute_result"
+                }
+            ],
+            "source": [
+                "brier_score_for_ensemble(fcst1, obs, event_thresholds=[-1, 0.5, 1, 3], ensemble_member_dim=\"ensemble\")"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "\n",
+                "\n",
+                "### Things to try next\n",
+                "- You can pass in a list of thresholds into the `threshold` arg if you want to calculate the Brier score for multiple thresholds.\n",
+                "- You can set `event_threshold_mode=operator.gt` if you don't want the exact threshold to be included as an event.\n",
+                "\n",
+                "### References\n",
+                "\n",
+                "- Brier, G.W., 1950. Verification of forecasts expressed in terms of probability. Monthly weather review, 78(1), 1-3. [https://doi.org/10.1175/1520-0493(1950)078<0001:VOFEIT>2.0.CO;2](https://doi.org/10.1175/1520-0493(1950)078<0001:VOFEIT>2.0.CO;2)\n",
```

Comment:
Even though this is the official DOI, it won't render properly in one or both of Read the Docs or JupyterLab (I can't remember which one it didn't render properly in) - because of the ludicrous ending of this DOI... The DOI Foundation has an official DOI shortening service (https://shortdoi.org/) which had already created a shortened (and sensible) DOI that does render properly. 

As such, in PR #751, @tennlee changed to the reference below (with the DOI that does render properly):

Reference**: Brier, G. W. (1950). Verification of forecasts expressed in terms of probability. Monthly weather review, 78(1), 1-3. [https://doi.org/fp62r6](https://doi.org/fp62r6)"

Reply:
Obviously neither Tennessee nor I knew creating the conflict would result in such difficulty, or we wouldn't have done it. So apologies for the negative impact.

Reply:
Ah okay - it's probably Read the Docs that it doesn't render. I'll update it to make it the same across both (the shorter version).

No problem - I never thought about what would happen if you had conflicts in a notebook